vim/load.ts

```diff
@@ -0,0 +1,63 @@
+import { Denops } from "../deps.ts";
+import { execute } from "./execute.ts";
+
+interface FileInfo {
+  isDirectory: boolean;
+}
+
+export async function load(
+  denops: Denops,
+  filePathInfo: URL,
+): Promise<void> {
+  const scriptPath = await setFile(denops, filePathInfo);
+
+  await execute(denops, `source ${scriptPath}`);
```

Comment:
*should*

1. Please consider what happen if `scriptPath` contains spaces, backslashes, or some special characters. Probably `scriptPath` should be escaped properly (prob. one of `fnameescape()` or `shellescape()` is good choice)
2. Please use third attribute of the `execute` function to pass `scriptPath` variable through context. It's a bit safer choice.

Reply:
> 1
Get out this error, if `scriptPath` contains spaces, exclamation mark and question mark.

```
Unexpected error occurred in 'loadfilesample' NotFound: File not found
at processResponse (deno:core/core.js:213:11)
at Object.jsonOpAsync (deno:core/core.js230:12)
at async Object.copyFile (deno:runtime/js/30_fs.js:50:5)
at async setFile (load.ts:25:5)                                                                                           
at async load (load.ts:12:22)                                                                                             
at async Vim.load (vim.ts:62:5)                                                                                           
at async mod.ts:18:5                                                                                                      
at async Promise.all (index 1) 
```

I will fix it, use to `shellescape()`.